src/parachain/rewards.ts

```diff
@@ -178,6 +178,39 @@ export class DefaultRewardsAPI implements RewardsAPI {
         );
     }
 
+    async getBlockRewards(
+        vaultAccountId: AccountId,
+        collateralCurrencyId: CollateralIdLiteral,
+        rewardCurrencyId: CurrencyIdLiteral,
```

Comment:
The reward currency is always the governance currency and I think we have a getter for that/define it globally for the lib. Hence, we could remove that parameter.

Reply:
Nice to have though since I think this is consistent with the current vault api.